dotnet/CacheLoaderAndRefresherSample/RefresherSample/LoaderAndRefresherSample.csproj

```diff
@@ -1,24 +1,18 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Alachisoft.NCache.SDK" Version="5.2.0" />
+    <PackageReference Include="Alachisoft.NCache.SDK" Version="5.2.0.1" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\SampleData\SampleData.csproj" />
   </ItemGroup>
 
   <ItemGroup>
-    <None Update="client.ncconf">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-    <None Update="config.ncconf">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
```

Comment:
Because if we install the SDK in a Solution containing multiple projects, the config files are only copied to one project and thus only that project has the references in its csproj file.